gcp/modules/couchdb/main.tf

```diff
@@ -84,7 +84,7 @@ resource "null_resource" "couchdb_finish_cluster" {
         if [ "$CLUSTER_MEMBERS_COUNT" != "${var.replica_count}" ]; then
           CLUSTER_READY="false"
         fi
-        if [ "$RETRY_COUNT" == "$RETRIES" ]; then
+        if [ "$RETRY_COUNT" == "$RETRIES" ] && [ "$CLUSTER_READY" != "true" ]; then
```

Comment:
Can we be consistent with the logic of checks:
here: `"$CLUSTER_READY" != "true"`
line 92: `"$CLUSTER_READY" == "false"`
main while loop: `"$CLUSTER_READY" != "true" `

Reply:
Sounds good!